### DISCUSSION TITLE
invisible text layer

### DISCUSSION THREAD

**Discussion by lbr991** [py-pdf/pypdf#1940] [Q&A]
I'm trying to take a non-searchable pdf and convert it to searchable pdf by superimposing an invisible text layer.
I need the invisible text bboxes to align exactly with the original bboxes (extracted by Textract).
I also don't want to specify a font and font_size because then the bboxes wouldn't align perfectly.

Is this possible with pypdf?

**Comment by pubpub-zz** [comment]
pypdf is not a library to create some pages. I would more likely recommend fpdf2. where you can set the text_mode to set the text invisible.
